torch/optim/lr_scheduler.py

```diff
@@ -365,7 +366,10 @@ def step(self, metrics, epoch=None):
         self.last_epoch = epoch
 
         if self.is_better(current, self.best):
-            self.best = current
+            if isinstance(current, torch.Tensor):
```

Comment:
Isn't `current` supposed to be a scalar?
If yes, can't you do something like
```python
self.best = float(current)
```
?